sdk/src/types/block/output/native_token.rs

```diff
@@ -20,7 +20,7 @@ use crate::types::block::{output::TokenId, Error};
 #[packable(unpack_error = Error)]
 pub struct NativeToken {
     // Identifier of the native token.
-    #[serde(rename = "id")]
+    #[cfg_attr(feature = "serde", serde(rename = "id"))]
```

Comment:
remove cfg_attr ?

Reply:
no 😅

Reply:
I thought it wasn't necessary when we already had it on the struct?

Reply:
Yeah I was wrong about that as I explained yesterday. I just didn't see it break because we always compile with serde

Reply:
Oh alright, yeah I see